starcounter-include.html

```diff
@@ -476,13 +519,25 @@
          * @param {DocumentFragment} givenComposition
          */
         StarcounterIncludePrototype.stampComposition = function(givenComposition){
-            // create shadowRoot if none
             const shadowRoot = this.shadowRoot;
+            this.blockingShadowLinksLoaded && this.blockingShadowLinksLoaded.reject();
 
             if (givenComposition !== undefined) {
+                this.setAttribute('loading-presentation', '');
```

Comment:
Can this be moved after the line 534? Then the else in that `if` won't be needed.

Reply:
:+1: Great idea.

Reply:
err...  after second thought, I'd say we still need the else. If the old composition is still being loaded. Then we stamp new one, which does not have any blocking links, we would like to remove attribute synchronously, not to have FONC (Flash Of No Content) - attribute removed on (even instantly) rejected promise will be removed asynchronously.

Reply:
but still we can move `setAttr..` to l.534,  as this is the only case and time we need it.